trapdata/api/tests/antenna_api_server.py

```
@@ -0,0 +1,188 @@
+
+app = FastAPI()
+
+# State management for tests
+_jobs_queue: dict[int, list[AntennaPipelineProcessingTask]] = {}
+_posted_results: dict[int, list[AntennaTaskResult]] = {}
+_projects: list[dict] = []
+_registered_pipelines: dict[int, list[str]] = {}  # project_id -> pipeline slugs
+
+
+@app.get("/api/v2/jobs")
+def get_jobs(pipeline__slug: str, ids_only: int, incomplete_only: int):
```

Ruff (0.14.14)

Unused function argument: `pipeline__slug`

(ARG001)